### PR TITLE
media-video/jellyfin-media-player: new package

### PR DESCRIPTION
**Pull request by ArsenArsen**

### CODE REVIEW

File: media-video/jellyfin-media-player/jellyfin-media-player-1.6.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{6..10} )
```
Comment (telans) [inline media-video/jellyfin-media-player/jellyfin-media-player-1.6.1.ebuild:6]:
```suggestion
PYTHON_COMPAT=( python3_{8..10} )
```

File: media-video/jellyfin-web-jmp-bin/jellyfin-web-jmp-bin-10.7.6.ebuild [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{6..10} )
```
Comment (telans) [inline media-video/jellyfin-web-jmp-bin/jellyfin-web-jmp-bin-10.7.6.ebuild:6]:
```suggestion
PYTHON_COMPAT=( python3_{8..10} )
```

File: media-video/jellyfin-media-player/metadata.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>arsen@aarsen.me</email>
+		<name>Arsen Arsenović</name>
+	</maintainer>
+	<maintainer type="project">
```
Comment (telans) [inline media-video/jellyfin-media-player/metadata.xml:8]: I'm the maintainer for this package in ::guru, would you mind adding me here too?

```
<email>telans@posteo.de</email>
<name>James Beddek</name>
```

File: media-video/jellyfin-media-player/jellyfin-media-player-1.6.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+PYTHON_COMPAT=( python3_{8..10} )
+inherit cmake python-any-r1
+
+DESCRIPTION="Jellyfin Desktop Client based on Plex Media Player"
+HOMEPAGE="https://github.com/jellyfin/jellyfin-media-player"
+LICENSE="GPL-2"
+SLOT="0"
+
+SRC_URI="
+	https://github.com/jellyfin/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
```
Comment (thesamesam) [inline media-video/jellyfin-media-player/jellyfin-media-player-1.6.1.ebuild:15]: Copy order from skel.ebuild

File: media-video/jellyfin-web-jmp-bin/jellyfin-web-jmp-bin-10.7.6.ebuild [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Modified Jellyfin Web Client for use inside Jellyfin Media Player."
+HOMEPAGE="https://github.com/jellyfin/jellyfin-media-player"
+LICENSE="GPL-2"
+SLOT="0"
```
Comment (thesamesam) [inline media-video/jellyfin-web-jmp-bin/jellyfin-web-jmp-bin-10.7.6.ebuild:9]: Ordrr

**Comment by ArsenArsen** [comment]
reordered

**Pull request closed without merging**